### ISSUE TITLE
stty: standard input: Inappropriate ioctl for device

### ISSUE THREAD

**Issue by jcassee** [sorin-ionescu/prezto#1770]
### Description
Since e86200e7aad9d4ccbabd34222ffcded3f52c6f65, prezto will execute `stty -ixon` on startup. On my system, this results in the following error:
```
stty: standard input: Inappropriate ioctl for device
```

It might be because I am using Powerline10k with its instant prompt. I think Standaard input and output is redirected if you use it.

I have had this in my `.zshrc` for a while, which does work:
```
stty -F/dev/tty -ixon -ixoff
```

However, I'm not sure `/dev/tty` is correct in all cases. Maybe add on option to disable the `stty` command or specify the tty explicitly?

### Expected behavior
No errors during the start of zsh.

### Actual behavior
The error: `stty: standard input: Inappropriate ioctl for device`

### Steps to Reproduce

1. Start zsh

### Versions

  - Prezto commit: e86200e7aad9d4ccbabd34222ffcded3f52c6f65
  - ZSH version: zsh 5.7.1 (x86_64-ubuntu-linux-gnu)
  - OS information: Ubuntu 19.10

**Comment by cyrinux** [comment]
Hi, to say iI got the same zsh stack as you and got the same error.

**Comment by jcassee** [comment]
Fixed by #1771

**Comment by belak** [comment]
Should be fixed now. Thanks for the report!
